Bottom: c2570d877e4b6274a84b76e14af49deef5eab0d6
Top:    12f62c20846673de10f7c899de38bc16574a78ad
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:18:06 +0000

Refresh of enable-output-of-via-pads-and

---

diff --git a/src/polygon.h b/src/polygon.h
index aabc096..15df8cc 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 10 //40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
